options/index.js

```diff
@@ -3,58 +3,30 @@ import ReactDOM from 'react-dom'
 
 import '../common/hot-reload'
 
-const Form = () => {
+const Options = () => {
     const [webhookUrl, setWebhookUrl] = useState('')
+    const handleChange = (e) => setWebhookUrl(e.target.value) // TODO: handle url validation
+    const handleClick = (e) => {
+        e.preventDefault()
```

Comment:
What does calling `preventDefault()` achieve? Is there some default behavior of the `<input>` field that we want to prevent?

Reply:
Yeah, the input element with type submit will refresh the browser by default and that's not necessary for us since we're handling the data in React state.